### PR TITLE
Feat/HNG 75 NEW LINK SENT EMAIL TEMPLATE

### PR DESCRIPTION
**Pull request by SAGHEDEV**
# Description

**Closes #issue_number_here**
#75 

# Changes proposed
I WORKED ON THE TEMPLATE FOR NOTIFYING THE USER THAT A NEW ACTIVATION LINK WAS SENT TO THEM

## What were you told to do?
I WAS TOLD TO CREATE A NEW LINK SENT EMAIL TEMPLATE

[Feature]: AUTHENTICATION -> EMAIL TEMPLATE -> New Link Sent Template #75

## What did you do?

I CREATED A FOLDER FOR THE ACTIVATE ACCOUNT AND ALSO A LAYOUT FOLDER FOR LAYING OUT THE TEMPLATE

ALSO I HAD TO MAKE MY HEADER AND FOOTER MYSELF SINCE I CAN'T SEEM TO SEE ANYBODY ASSIGNED TO IT, WHICH I ADDED TO THE COMPONENT FOLDER

THEN I CREATED TWO FILES IN THE ACTIVATE ACCOUNT FOLDER FOR THE TEMPLATE WITH IMAGE ADDED AND THE ONE WITHOUT IMAGE.

I USED REACT EMAKIL INSTALLED ON THE REPO TO CREATE THE TEMPLATE

AND I HAD TO HARD CODE THE COLORS SINCE THEY WERE NOT YET PROVIDED ON THE REPO YET

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [X ] My code follows the code style of this project.
- [X ] This PR does not contain plagiarized content.
- [X ] The title and description of the PR is clear and explains the approach.
- [X ] I am making a pull request against the **dev branch** (left side).
- [X ] My commit messages styles matches our requested structure.
- [X ] My code additions will fail neither code linting checks nor unit test.
- [X ] I am only making changes to files I was requested to.

# Screenshots/Videos

Screenshot 2024-07-21 095239
Screenshot 2024-07-21 095209
Screenshot 2024-07-21 095141
Screenshot 2024-07-21 095126
Screenshot 2024-07-21 095100
Screenshot 2024-07-21 095045

### CODE REVIEW

File: app/components/ui/switch.tsx [hunk 1]
```diff
@@ -2,15 +2,11 @@
 
 import * as SwitchPrimitives from "@radix-ui/react-switch";
 import { cn } from "app/lib/utils/cn";
-import {
-  forwardRef,
-  type ComponentPropsWithoutRef,
-  type ElementRef,
-} from "react";
+import * as React from "react";
```
Comment (chumex412) [inline app/components/ui/switch.tsx:5]: Revert these boss, import only what you need from React

Comment (chumex412) [reply app/components/ui/switch.tsx:5]: Don't import all of React

Comment (chumex412) [reply app/components/ui/switch.tsx:5]: Don't import all of React

File: app/email/templates/_components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const Header = () => {
+  return (
+    <div className="bg-[#E1D6D6] p-[46px]">
+      <div className="flex w-full items-center justify-center py-[46px]">
+        <svg
```
Comment (chumex412) [inline app/email/templates/_components/Header.tsx:5]: Please look for a better way to use this SVG icon, using the markup is not standard practice

File: app/email/templates/activate-account/no-image.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import { Button } from "@react-email/components";
+
+import TailwindWrapper from "../_components/tailwindWrapper";
+import Layout from "../Layouts/Layout";
+
+interface propertyDataType {
```
Comment (chumex412) [inline app/email/templates/activate-account/no-image.tsx:6]: That's not a proper interface naming convention

File: app/email/templates/activate-account/with-image.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import { Button } from "@react-email/components";
+
+import TailwindWrapper from "../_components/tailwindWrapper";
+import HeroIcon from "../icons/heroIcon";
+import Layout from "../Layouts/Layout";
+
+interface propertyDataType {
```
Comment (chumex412) [inline app/email/templates/activate-account/with-image.tsx:7]: Same here, use PascalCase

File: app/email/templates/icons/heroIcon.tsx [hunk 5]
```diff
@@ -0,0 +1,186 @@
+const HeroIcon = () => {
+  return (
+    <div className="flex items-center justify-center">
+      <div className="hidden lg:block">
```
Comment (chumex412) [inline app/email/templates/icons/heroIcon.tsx:4]: Please all assets should be in the public folder. Find a better way to include svg icons.

**Review by chumex412 — CHANGES_REQUESTED** [review]

File: app/components/ui/switch.tsx (hunk 1 above)
Comment (SAGHEDEV) [reply app/components/ui/switch.tsx:5]: Though this isn't my component sir

Comment (SAGHEDEV) [reply app/components/ui/switch.tsx:5]: Should I make the changes?

**Review by shalomtaiwo — CHANGES_REQUESTED** [review]
Please check requested changes from the previous mentor

**Review by mrcoded — APPROVED** [review]

**Pull request merged**